src/Web3.elm

```diff
@@ -186,17 +190,133 @@ toChecksumAddress (Address address) =
         }
 
 
+toWei : EthUnit -> String -> Result String BigInt
+toWei unit amount =
+    -- check to make sure input string is formatted correctly, should never error in here.
+    if Regex.contains (Regex.regex "^\\d*\\.?\\d+$") amount then
+        let
+            decimalPoints =
+                decimalShift unit
 
-{-
-   TODO
-   fromWei : EthUnit -> BigInt -> String
+            formatMantissa =
+                String.slice 0 decimalPoints >> String.padRight decimalPoints '0'
 
-   toWei : EthUnit -> String -> Maybe BigInt
+            finalResult =
+                case (String.split "." amount) of
+                    [ a, b ] ->
+                        a ++ (formatMantissa b)
 
-   bigIntToWei : EthUnit -> BigInt -> BigInt
```

Comment:
I see I left this one off. Want me to add it again?

Reply:
Sure! Thanks